modules/backend/common/enumerations/sex/Sex.cpp

```diff
@@ -21,7 +11,7 @@ std::string toString(const Sex& sex)
     case Sex::Female:
         return "Female";
     default:
-        return;
+        return {"Unknown"};
```

Comment:
Such situation will never occur if Pesel will be responsible for defining Sex, we can simply keep 'return;'

Reply:
Or if it's necessary add Unknown as a state of enum type and some convertion layer that will change values different than Female/Male to Unknown

Reply:
Done